bpf-gpl/xdp.c

```diff
@@ -65,11 +66,63 @@ int calico_xdp(struct xdp_md *xdp_ctx) {
 	}
 
 	// Parse packets and drop malformed and unsupported ones
-	if (parse_packet_ip(&ctx) == -2) {
-		return XDP_DROP;
+	switch (parse_packet_ip(&ctx)) {
+	case -1:
```

Comment:
I think we should #define constants for these -1 and -2 values, so it is easier to see what they mean.

Reply:
Looks like -1 = DROP, -2 = ALLOW_WITHOUT_ENFORCING_POLICY , is that right?

Reply:
Exactly, will define the constants.